arelle/CntlrCmdLine.py

```diff
@@ -579,11 +578,27 @@ class CntlrCmdLine(Cntlr.Cntlr):
     Initialization sets up for platform via Cntlr.Cntlr.
     """
 
-    def __init__(self, logFileName=None, uiLang=None, disable_persistent_config=False):
-        super(CntlrCmdLine, self).__init__(hasGui=False, uiLang=uiLang, disable_persistent_config=disable_persistent_config)
-        self.preloadedPlugins =  {}
+    def __init__(
+        self,
+        options: RuntimeOptions | None = None,
```

Comment:
It would be better to append options as the last parameter so the prior constructor signature is still valid.